xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/test/java/org/xwiki/rendering/internal/macro/DefaultMacroCategoryManagerTest.java

```diff
@@ -19,122 +19,181 @@
  */
 package org.xwiki.rendering.internal.macro;
 
+import java.util.HashSet;
+import java.util.Properties;
 import java.util.Set;
 
-import org.jmock.Expectations;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.xwiki.rendering.internal.transformation.macro.DefaultMacroTransformationConfiguration;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 import org.xwiki.rendering.macro.Macro;
 import org.xwiki.rendering.macro.MacroCategoryManager;
 import org.xwiki.rendering.macro.MacroId;
+import org.xwiki.rendering.macro.MacroLookupException;
+import org.xwiki.rendering.macro.MacroManager;
 import org.xwiki.rendering.macro.descriptor.DefaultMacroDescriptor;
-import org.xwiki.rendering.syntax.Syntax;
 import org.xwiki.rendering.transformation.macro.MacroTransformationConfiguration;
-import org.xwiki.test.jmock.AbstractComponentTestCase;
+import org.xwiki.test.LogLevel;
+import org.xwiki.test.junit5.LogCaptureExtension;
+import org.xwiki.test.junit5.mockito.ComponentTest;
+import org.xwiki.test.junit5.mockito.InjectMockComponents;
+import org.xwiki.test.junit5.mockito.MockComponent;
+
+import ch.qos.logback.classic.Level;
```

Comment:
Is this really the correct import?

Reply:
Yes, that's the type returned by `ch.qos.logback.classic.spi.ILoggingEvent#getLevel` (`ch.qos.logback.classic.Level`)